odoo/tools/mail.py

```diff
@@ -46,7 +46,7 @@ def getaddresses(fieldvalues):
      'data-o-mail-quote', 'data-o-mail-quote-node',  # quote detection
      'data-oe-model', 'data-oe-id', 'data-oe-field', 'data-oe-type', 'data-oe-expression', 'data-oe-translation-initial-sha', 'data-oe-nodeid',
      'data-last-history-steps', 'data-oe-protected', 'data-oe-transient-content', 'data-width', 'data-height', 'data-scale-x', 'data-scale-y', 'data-x', 'data-y',
-     'data-publish', 'data-id', 'data-res_id', 'data-interval', 'data-member_id', 'data-scroll-background-ratio', 'data-view-id',
+     'data-publish', 'data-id', 'data-res_id', 'data-interval', 'data-member_id', 'data-scroll-background-ratio', 'data-view-id', 'data-oe-t-inline',
```

Comment:
I feel like there will be more repercussion than you think if you change this file.
[Here](https://github.com/odoo/odoo/blob/2b55e263bd51ca24724100d71f4d9a3866e16c8b/addons/html_editor/static/src/scss/html_editor.common.scss#L919-L924) lies the style that applies the problematic issue.

I also feel like you are going a bit too far with the actual issue of the task.
In the task it is said to only change this in mass_mailing, was there some discussion to change it to a more generic fix?